Spotless check requires main branch to be cloned

Building any branch with spotless requires the main branch to be checked out.  This is problematic when running from CI.

```
> Configure project :
pa in dir: (../performance-analyzer) will be built.
PA repo located at '../performance-analyzer' will be used.

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'rca'.
> No such reference 'origin/main'


```

--- a/build.gradle
+++ b/build.gradle
@@ -96,9 +96,6 @@ ext {
 
 spotless {
     java {
-        // only apply formatting rules to updated files
-        ratchetFrom "origin/${gitDefaultBranch}"
-
         licenseHeaderFile(file('license-header'))
         googleJavaFormat().aosp()
         importOrder()

